plugins/drush/README.md

```diff
@@ -0,0 +1,31 @@
+# Drush plugin #
+
+## Description ##
+This plugin offers aliases and functions to make the work with drush easier and more productive.
+
+## How to use ##
```

Comment:
This whole section should be restructured to use markdown tables. [See the `xcode` plugin for an example](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/xcode/README.md#aliases).